### ISSUE TITLE
Slider doesn't work well with combination of mouse and keyboard

### ISSUE THREAD

**Issue by jessegreenberg** [phetsims/john-travoltage#284]
Possibly introduced during #271, but I noticed that if I drag an appendage to a particular location then continue to interact with the keyboard, the appendage jumps to the wrong location. @mbarlow12 can you look into this? Let me know if the AccessibleSlider trait makes fixing this difficult.

**Comment by mbarlow12** [comment]
@jessegreenberg I think this may be a result of how we've implemented accessible slider. We've created an intermediate  `inputValueProperty` that is passed to the slider. It simultaneously listens to changes in the `angleProperty` while also notifying `angleProperty` of changes in itself. As a result, both properties reupdate one another several times.

Would it be possible to mandate that AccessibleSlider always updates the specified property (likely from the model) directly, then allows for some kind of range or linear mapping option for handling updates in the PDOM? Handling keyboard input might be a little tricky, but I think we should consider ways to prevent this kind of circular information flow.

**Comment by jessegreenberg** [comment]
@mbarlow12 and I talked about this today, we discussed using trying the AccessibleSlider option `accessibleMapValue`.

**Comment by mbarlow12** [comment]
Closing. Remaining related issue is documented in #285 .
